core/src/androidMain/kotlin/com/datadog/kmp/Datadog.kt

```diff
@@ -15,7 +15,7 @@ import com.datadog.kmp.core.configuration.setProxy
 import com.datadog.kmp.internal.InternalAttributes
 import com.datadog.kmp.privacy.TrackingConsent
 import android.util.Log as AndroidLog
-import com.datadog.android.Datadog as DatadogAndroid
+import com.datadog.android.Datadog as DatadogAndroidImpl
```

Comment:
what is the added value of such rename? This may create a false assumption that there is `DatadogAndroid` interface somewhere, but this is not true.

Reply:
no reason, just debug leftover